.github/workflows/pr_test_cpu.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false # TODO: enable it when CI be stable
       matrix:
         os: ['Ubuntu', 'Windows', 'MacOS']
-        python-version: ['3.7', '3.8']
+        python-version: ['3.7', '3.10']
```

Comment:
Torch just support py310 after `1.11.0` -- https://github.com/kornia/kornia/actions/runs/3867533786/jobs/6592374161#step:3:577

Reply:
i added the 3.9 for the older version, and exclude the unsupported cases